src/TextInput/TextInput.docs.json

```diff
@@ -128,7 +128,7 @@
         {
           "name": "tooltipDirection",
           "type": "'n' | 'ne' | 'e' | 'se' | 's' | 'sw' | 'w' | 'nw'",
-          "defaultValue": "n",
+          "defaultValue": "s",
```

Comment:
This might be the default on `Tooltip2`, but I think we should explicitly set `'s'` as the default value of `tooltipDirection` in the `TextInput.Action` component.

This way the docs will always be accurate, even if we decide to change the default direction in `Tooltip2`.

Reply:
>This might be the default on Tooltip2, but I think we should explicitly set 's' as the default value of tooltipDirection in the TextInput.Action component.

No worries, that sounds good. I'll update it now.